Description: Fix udev rule for hid2hci and Logitech dongles doesn't work with kernels >= 3.6
Origin: http://www.spinics.net/lists/linux-bluetooth/msg31851.html
Author: Alexander Holler <holler@ahsoftware.de>

From d557e0d956d69318664ed5d1c3e4a765da24bc2a Mon Sep 17 00:00:00 2001
From: Alexander Holler <holler@ahsoftware.de>
Date: Tue, 20 Nov 2012 12:47:08 +0100
Subject: [PATCH] hid2hci: change subsystem in udev rule from usb to usb*

With kernel 3.6 (commit 7e97243c2080ecae7129e83635227fdebd4feef6) the
class for some devices (e.g. dongles from Logitech) were changed from
usb to usbmisc. As consequence the udev rule for hid2hci didn't work
anymore with kernels >= 3.6.

Changing the subsystem from "usb" to "usb*" matches both "usb" and
"usbmisc" and works with all kernels.
---
 scripts/bluetooth-hid2hci.rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/bluetooth-hid2hci.rules b/scripts/bluetooth-hid2hci.rules
index 0687c8a..db6bb03 100644
--- a/scripts/bluetooth-hid2hci.rules
+++ b/scripts/bluetooth-hid2hci.rules
@@ -1,7 +1,7 @@
 # do not edit this file, it will be overwritten on update
 
 ACTION=="remove", GOTO="hid2hci_end"
-SUBSYSTEM!="usb", GOTO="hid2hci_end"
+SUBSYSTEM!="usb*", GOTO="hid2hci_end"
 
 # Variety of Dell Bluetooth devices - match on a mouse device that is
 # self powered and where a HID report needs to be sent to switch modes
-- 
1.8.0

